src/react-components/stories/1-Button.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-import Button from "../components/Button/Button";
+import Button from "../src/components/Button/Button";
```

Comment:
I think it might be better to mimic the Twig structure since it's the same structure as in Metronome, right?  Also could take this time to move stories to be in the same folder as the tsx.

Reply:
I think it's best to stay consistent across our three packages. It's a lot of folders, but at least you always know where you're going instead of `find . --name`ing for files.